backend/use_cases/bias_calculators/bias_calculator.py

```diff
@@ -48,92 +38,88 @@ class BiasCalculator:
 
     def calculate_overall_score(self, df, categories: set):
         """
-        Calculate an overall bias score across multiple categories in a DataFrame.
-        The score is calculated by averaging scores for each category based on the specified method.
+    Calculate an overall bias score across multiple categories in a DataFrame.
+
+    This method computes an average score for the specified categories, where each
+    category's score is based on the variance of false positive rates (FPRs).
 
-        Parameters:
-        df (pd.DataFrame): The DataFrame containing the infrastructure.
+    Args:
+        df (pd.DataFrame): The DataFrame containing the infrastructure data.
         categories (set): A set of column names representing the categories to calculate scores for.
-        method (str): The method used to calculate individual category scores. Options are:
-                      - "variance": Uses variance of FPRs for each category.
-                      - "fpr_mean": Uses mean of FPRs for each category.
-                      - "accuracy": (Optional) Placeholder for calculating based on accuracy.
 
-        Returns:
+    Returns:
         float: An average score between 0 and 10, where higher values indicate lower bias across categories.
 
-        Notes:
-        - The function averages scores calculated for each category, so `calculate_score_by_variance`
-          and `calculate_score_by_fpr_mean` should be defined separately.
-        - By default, the "variance" method is used. If "accuracy" is selected, it will currently return 0
-          until `calculate_score_by_accuracy` is implemented.
+    Raises:
+        NotImplementedError: This method is not yet implemented.
+
+    Note:
+        - The function averages scores calculated for each category.
         - Assumes each scoring method returns a score between 0 and 10.
-        """
+    """
         raise NotImplementedError
 
     def calculate_score(self, df, category):
         """
-        Calculate a bias score based on the variance of false positive rates (FPRs) for a given category.
-        The score ranges from 0 to 10, with a lower FPR variance resulting in a higher score.
+     Calculate a bias score based on the variance of false positive rates (FPRs) for a given category.
```

Comment:
Formatting seems off here, the comment should start at the same indent as the """